dot/rpc/websocket.go

```diff
@@ -121,17 +139,22 @@ func NewWSConn(conn *websocket.Conn, cfg *HTTPServerConfig) *WSConn {
 		storageSubChannels: make(map[int]byte),
 		storageAPI:         cfg.StorageAPI,
 		blockAPI:           cfg.BlockAPI,
+		runtimeAPI:         cfg.RuntimeAPI,
+		coreAPI:            cfg.CoreAPI,
+		txStateAPI:         cfg.TransactionQueueAPI,
 	}
 	return c
 }
 
-func (c *WSConn) safeSend(msg interface{}) error {
+func (c *WSConn) safeSend(msg interface{}) {
 	c.mu.Lock()
 	defer c.mu.Unlock()
-
-	return c.wsconn.WriteJSON(msg)
+	err := c.wsconn.WriteJSON(msg)
+	if err != nil {
+		logger.Warn("error sending websocket message", "error", err)
```

Comment:
I wonder if this should be warn or debug, since warn will pollute the logs if there's some issue... what do you think?

Reply:
Good idea.